.circleci/config.yml

```diff
@@ -208,7 +208,7 @@ workflows:
             - prep-build-test-mv2
             - get-changed-files-with-git-diff
       - test-e2e-firefox-confirmation-redesign:
-          <<: *develop_master_rc_only
+          # <<: *develop_master_rc_only
```

Comment:
Is this change intended to be part of this PR?

Reply:
This is just to test the fix addresses the problem on CI, as per [this conversation](https://consensys.slack.com/archives/CTQAGKY5V/p1721222766599959?thread_ts=1721211621.814389&cid=CTQAGKY5V).

I added the `DO-NOT-MERGE` label to the PR and will remove the change before merging.